core/src/main/java/org/bitcoinj/core/PeerAddress.java

```diff
@@ -351,10 +355,17 @@ public BigInteger getServices() {
         return services;
     }
 
-    public long getTime() {
+    @Nullable
```

Comment:
We should definitely avoid this `@Nullable`, perhaps by using an `Optional<Duration>`?

Reply:
Ok, I changed this to Optional.